ament_tools/verbs/build/cli.py

```diff
@@ -114,13 +114,13 @@ def prepare_arguments(parser, args):
         help='End with a particular package',
     )
     parser.add_argument(
-        '--only-package',
-        '--only',
```

Comment:
I agree, but it won't be displayed in the help message anymore if it is removed here.